backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/CassandraTableManager.java

```diff
@@ -59,7 +59,7 @@ public InitializationStatus initializeTables() {
     public void clearAllTables() {
         CassandraAsyncExecutor executor = new CassandraAsyncExecutor(session);
         Flux.fromIterable(module.moduleTables())
-                .publishOn(Schedulers.elastic())
+                .publishOn(Schedulers.boundedElastic())
```

Comment:
What's the bound? Can I configure it? Maybe I'd be happy to have some documentation about such an important parameter....

Reply:
The documentation says ```it has a cap on the number of backing threads it can create (default is number of CPU cores x 10```

The default size is : 

```
	public static final int DEFAULT_BOUNDED_ELASTIC_SIZE =
			Optional.ofNullable(System.getProperty("reactor.schedulers.defaultBoundedElasticSize"))
					.map(Integer::parseInt)
					.orElseGet(() -> 10 * Runtime.getRuntime().availableProcessors());
```

So it can be configured by a property, otherwise there are constructors which allow to configure the size and queue size.

Reply:
https://projectreactor.io/docs/core/release/api/reactor/core/scheduler/Schedulers.html#DEFAULT_BOUNDED_ELASTIC_SIZE

> Default maximum size for the global boundedElastic() Scheduler, initialized by system property reactor.schedulers.defaultBoundedElasticSize and falls back to 10 x number of processors available to the runtime on init.

Unbounded elastic schedulers will be deprecated.